### DISCUSSION TITLE
Erro na instalação do pacote do EntityFramework

### DISCUSSION THREAD

**Discussion by RonaldoBueno16** [balta-io/forum#386] [Tira Dúvidas] [answered]
Balta, bom dia, tudo bem?

Estou fazendo o curso de "Fundamentos ASP NET 6" e estou tendo o seguinte problema na intalação do pacote do EntityFramework Sqlite:

`dotnet add package Microsoft.EntityFrameworkCore.SqlServer
  Determinando os projetos a serem restaurados...
  Writing C:\Users\ronaldo.junior\AppData\Local\Temp\tmp795F.tmp
info : X.509 certificate chain validation will use the default trust store selected by .NET.
info : X.509 certificate chain validation will use the default trust store selected by .NET.
info : Adicionando PackageReference do pacote 'Microsoft.EntityFrameworkCore.SqlServer' ao projeto 'C:\Users\ronaldo.junior\Desktop\Estudos\C#\T
odo\Todo.csproj'.
info :   GET https://pkgs.dev.azure.com/uipath/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/1c781268-d43d-45ab-9dfc-0151a1c740b7/nuget/v3/reg
istrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json
info :   GET https://uipath.pkgs.visualstudio.com/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/a98dba69-6b77-4e3a-9434-1f5e5addca74/nuget/v3/
registrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json
info :   NotFound https://pkgs.dev.azure.com/uipath/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/1c781268-d43d-45ab-9dfc-0151a1c740b7/nuget/v
3/registrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json 1434 ms
info :   NotFound https://uipath.pkgs.visualstudio.com/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/a98dba69-6b77-4e3a-9434-1f5e5addca74/nuge
t/v3/registrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json 1563 ms
info :   GET https://pkgs.dev.azure.com/uipath/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/1c781268-d43d-45ab-9dfc-0151a1c740b7/nuget/v3/reg
istrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json
info :   GET https://uipath.pkgs.visualstudio.com/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/a98dba69-6b77-4e3a-9434-1f5e5addca74/nuget/v3/
registrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json
info :   NotFound https://pkgs.dev.azure.com/uipath/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/1c781268-d43d-45ab-9dfc-0151a1c740b7/nuget/v
3/registrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json 359 ms
info :   NotFound https://uipath.pkgs.visualstudio.com/5b98d55c-1b14-4a03-893f-7a59746f1246/_packaging/a98dba69-6b77-4e3a-9434-1f5e5addca74/nuge
t/v3/registrations2-semver2/microsoft.entityframeworkcore.sqlserver/index.json 476 ms
error: Não há versões disponíveis do pacote 'Microsoft.EntityFrameworkCore.SqlServer'.

Usage: NuGet.CommandLine.XPlat.dll package add [options]

Options:
  -h|--help               Show help information
  --force-english-output  Força o aplicativo a ser executado usando uma cultura invariável baseada no inglês.
  -s|--source             Especifica as origens de pacote NuGet a serem usadas durante a restauração.
  --package-directory     Diretório no qual os pacotes devem ser restaurados.
  --interactive           Permitir que o comando bloqueie e exija ação manual para operações como a autenticação.
  --prerelease            Permite a instalação de pacotes de pré-lançamento.`

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @RonaldoBueno16, como vai?

Sugiro limpar seu cache local:
`nuget locals all -clear`

Vale a pena regerar os certificados http também:
```
dotnet dev-certs https --clean
dotnet dev-certs https --trust
```

Depois só tentar instalar novamente
